### ISSUE TITLE
[Feature Request] Update JSHandler to Implement IJSInProcessRuntime for WebAssembly

### ISSUE THREAD

**Issue by gt-jkinter** [apexcharts/Blazor-ApexCharts#307]
Hello - I'd like to start by thanking you for creating this wrapper, it's a fantastic implementation. 

I created this "issue" to propose a feature request to implement the use of `IJSInProcessRuntime` when using WebAssembly. 
The goal would be to eek out a bit more performance by avoiding the async overhead within IJSRuntime and using the synchronous JSInterop within WASM. 

Here's a small example of what it might look like within `RenderChartAsync()`: 

```
public async Task RenderChartAsync()
{
    if (JSRuntime is IJSInProcessRuntime jsInProcessRuntime)
    {
        jsInProcessRuntime.InvokeVoid("blazor_apexchart.renderChart", ObjectReference, ChartContainer, JsonSerializer.Serialize(ChartReference.Options, ChartSerializer.GetOptions<TItem>()), new JSEvents()
        {
            HasDataPointSelection = ChartReference.OnDataPointSelection.HasDelegate,
            HasDataPointEnter = ChartReference.OnDataPointEnter.HasDelegate || ChartReference.ApexPointTooltip != null,
            HasDataPointLeave = ChartReference.OnDataPointLeave.HasDelegate,
            HasLegendClick = ChartReference.OnLegendClicked.HasDelegate,
            HasMarkerClick = ChartReference.OnMarkerClick.HasDelegate,
            HasXAxisLabelClick = ChartReference.OnXAxisLabelClick.HasDelegate,
            HasSelection = ChartReference.OnSelection.HasDelegate,
            HasBrushScrolled = ChartReference.OnBrushScrolled.HasDelegate,
            HasZoomed = ChartReference.OnZoomed.HasDelegate,
            HasAnimationEnd = ChartReference.OnAnimationEnd.HasDelegate,
            HasBeforeMount = ChartReference.OnBeforeMount.HasDelegate,
            HasMounted = ChartReference.OnMounted.HasDelegate,
            HasUpdated = ChartReference.OnUpdated.HasDelegate,
            HasMouseLeave = ChartReference.OnMouseLeave.HasDelegate,
            HasMouseMove = ChartReference.OnMouseMove.HasDelegate,
            HasClick = ChartReference.OnClick.HasDelegate,
            HasBeforeZoom = ChartReference.OnBeforeZoom != null,
            HasBeforeResetZoom = ChartReference.OnBeforeResetZoom != null,
            HasScrolled = ChartReference.OnScrolled.HasDelegate
        });
    }
...
}
```
However, I'm unsure if this would be the **best** way to perform this implementation.

**Comment by joadan** [comment]
Hello,

I have done some experiments with this and I had a rough implementation but I could not see a performance  increase that would justify the added complexity.

Maybe I missed something, do you have some info what performance increase we should expect?

**Comment by gt-jkinter** [comment]
Unfortunately no, I've only seen [this article](https://www.meziantou.net/optimizing-js-interop-in-a-blazor-webassembly-application.htm) that discusses the performance implications. 

In my specific use case, we have a WASM application that sometimes renders up to 100-200 charts. I've already implemented virtualization, and performance isn't **_terrible_**, but I was hoping to squeeze out as much as possible.

**Comment by joadan** [comment]
Hi, i guess for 100-200 charts this would matter.

Please note that I only tested Sync vs Async jsinterop. 
I felt that Unmarshalled was a bit to much 'experimental' especially because I have no experience with it.  

I'm not aware that ApexRangeSeries would take more time to serialize, my guess is that the difference is coming from somewhere else.

Do you 'just' render the charts or are you also running methods, like UpdateSeries or UpdateOptions?

**Comment by gt-jkinter** [comment]
I'm not running any methods, such as `UpdateOptionsAsync` or `UpdateSeriesAsync`. However, the chart is nested inside of a parent component that receives the data as a parameter. When the parent component is initialized, I am setting the `_chartOptions.Xaxis.Min' and '_chartOptions.Xaxis.Max'.

Within this parent component, I'm also making use of the `OnRendered` event handler from the `ApexChart` component to switch a boolean value when the chart has finished rendering.

**Comment by joadan** [comment]
Fixed in v2.0.0
